### DISCUSSION TITLE
Meshcentral Desktop App

### DISCUSSION THREAD

**Discussion by tobias-carlbom** [Ylianst/MeshCentral#6055] [Q&A]
Hi,

I recently started using Meshcentral and i am super impressed by its functionallity! Awesome work :)

Would someone be interested in a desktop app for meshcentral? I have created a simple Electron.js project which uses the web-view component and renders the web ui of Meshcentral in a desktop app for MacOS and Windows.

//Tobias

**Comment by si458** [comment]
do share! it would be nice to see what the community is using meshcentral with/for!
we already sort of have a desktop app, but its windows only, and sort of works on mac/linux with wine
https://github.com/Ylianst/MeshCentralRouter

**Comment by tobias-carlbom** [comment]
Here's a link https://github.com/manprinsen/meshcentral-desktop-app

**Comment by PathfinderNetworks** [comment]
Just as an aside to this, you can do the same using the current version of Microsoft Edge.  If you open your MeshCentral site in Edge it will/should allow you to quickly and easily create the desktop app version with just a click or two.  I've been using it for a while now and it's pretty nice - mostly because it gives you a larger workspace and is a direct link from the desktop shortcut. Nothing extra to install or configure in Mesh. 
https://support.microsoft.com/en-us/topic/install-manage-or-uninstall-apps-in-microsoft-edge-0c156575-a94a-45e4-a54f-3a84846f6113

**Reply by si458** [reply]
beat me to it! its the same with Google Chrome,
visit your meshcentral web ui, then click the INSTALL button that appears in the address bar.
but its still nice to see people trying different ways to create desktop apps tho!
